### ISSUE TITLE
Database config for test env missing DATABASE_HOST

### ISSUE THREAD

**Issue by joenas** [huginn/huginn#1979]
Is there a particular reason why there is no `host: <%= ENV['DATABASE_HOST'] || "" %>` for the test environment? 

I wanted to get my dev environment set up with Docker, which proved to be... difficult. Mysql via Docker is a pain if you don't wanna link containers (I run the server locally and just the database in Docker). So I tried with `postgres` and to use that I had to add the above to `database.yml`.

I also tried using [develop.yml](https://github.com/cantino/huginn/blob/master/docker/single-process/develop.yml) for `docker-compose` but I couldn't get the test suite to run. Inside the container, there are no `spec` dir due to `.dockerignore` and outside the mysql db wasnt reachable. Unless I did something wrong I guess 😄

**Comment by dsander** [comment]
Hi @joenas,

I don't think there is a reason why the `host` is not configurable in `database.yml` for the tests. My guess is that until now nobody ran the tests with an external database 😄 .

The `develop.yml` compose file is meant for development on the Docker images itself not for development on Huginn (even though it will run with the changes made to the working directory). You can not run the specs with Docker at the moment, the images are build to run Huginn in the production environment and do not include the development and test dependencies.

We have a open pull request to use Vagrant for development and testing (#1958).

**Comment by joenas** [comment]
> The develop.yml compose file is meant for development on the Docker images itself 

That explains it 😃 I'll try and see if I can get a PR together for dev/testing with Docker! Maybe separate `docker-compose.yml` for Postgres and MySQL?

**Comment by joenas** [comment]
It's actually just the additions to `database.yml` that are needed. Should I commit example files for `docker-compose` to the root of the project and updated the readme with instructions or just create an entry in the Wiki?

**Comment by dsander** [comment]
Does the compose configuration differ a lot from the ones we already have? If you want to put it in the repository I think it should be inside the `docker` folder.

**Comment by joenas** [comment]
Well basically, since I just run the database, this is it:

```yaml
version: '2'
services:
  db:
    image: postgres:9.3.11
    ports:
      - "5432:5432"
```

Like you said it's not possible to run specs in the rails-container so I figured I'd just share my setup for people who want to develop but not install MySQL/Postgres or pollute their current databases. 

I figure an entry in the Wiki should be enough tho!

**Comment by dsander** [comment]
I see, yeah a Wiki entry sounds reasonable.

**Comment by joenas** [comment]
Fixed in #2002?

**Comment by dsander** [comment]
Yes looks like it 😄
